massa-executed-ops/src/ops_changes.rs

```diff
@@ -98,13 +108,20 @@ impl Deserializer<ExecutedOpsChanges> for ExecutedOpsChangesDeserializer {
                     context("operation id", |input| {
                         self.operation_id_deserializer.deserialize(input)
                     }),
+                    context("operation execution status", |input| {
```

Comment:
It's possible here to make directly a sub tuple in the tuple parser above. (Optional)